spec/http_request_spec.rb

```diff
@@ -77,6 +77,57 @@
           ["can't be blank", "is not included in the list"]
         )
       end
+
+      describe "source_ip" do
+        let(:address)      { '127.0.0.1' }
+        let(:ipv4_address) { '93.184.216.34' }
+        let(:ipv6_address) { '2606:2800:220:1:248:1893:25c8:1946' }
+
+        it "must not require an address" do
+          ip_address = described_class.new
```

Comment:
Rename `ip_address` to `http_request` so it's not confusing. We're testing `HTTPRequest` not `IPAddress`.